config/example/config.toml

```diff
@@ -116,7 +120,7 @@ admin_rank_colour_map = [
 	{ name = "Community Manager", colour = "#e91e63" },
 	{ name = "Game Admin", colour = "#238afa" },
 	{ name = "Trial Admin", colour = "#7fb6fc" },
-	{ name = "PR Reviewer", colour = "#c27c0e" },
+	{ name = "Developer", colour = "#c27c0e" },
```

Comment:
Can we change the color of this to what the dev say color (green color) is? It already looks to similar to patreon colors.

How will colors and permissions behave if you are both a mentor and a developer?

Reply:
Dev takes priority over mentor

Reply:
Also yes let me change that colour